<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="res_Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>591</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Сравнить</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>591</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPlainTextEdit" name="Text1"/>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="Text2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>591</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="text2lbl">
      <property name="text">
       <string>Текст 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="text1lbl">
      <property name="text">
       <string>Текст 2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDoubleSpinBox" name="porog">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>461</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_por">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Порог срабатывания</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
